frontend/src/components/Tournament.js

```diff
@@ -24,6 +25,51 @@ import {
 } from "../skeletons/Standings";
 import Breadcrumbs from "./Breadcrumbs";
 
+const TeamSeedingChange = props => {
+  const filteredByTeamID = Object.entries(
```

Comment:
Hmm so here we will be doing O(n^2) operations right.. Cause for every team we will be looping and filtering..

Can we reduce this to O(n) by having a global teams map storing current and initial seed and we can use that for O(1) fetching and showing in this component?

This will help with better page load timings as well I think

Reply:
done